recipes/cityhash/meta.yaml

```diff
@@ -0,0 +1,42 @@
+
+{% set name = "python-cityhash" %}
+{% set version = "0.2.3.post9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/c/cityhash/cityhash-{{ version }}.tar.gz
+  sha256: aa8919c00e8d8699df42dd788eda4f24ad13086ea9d83407fd7da4a0206d9d18
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<35]
```

Comment:
FWIW we've dropped all Python versions pre-3.5. It may be worth skipping Windows and trying again in the feedstock.